skimage/transform/_warps.py

```diff
@@ -25,7 +25,7 @@ def resize(image, output_shape, order=None, mode='reflect', cval=0, clip=True,
 
     Performs interpolation to up-size or down-size N-dimensional images. Note
     that anti-aliasing should be enabled when down-sizing images to avoid
-    aliasing artifacts. For down-sampling with an integer factor also see
+    aliasing artifacts. For down-sizing with an integer factor also see
```

Comment:
```suggestion
    aliasing artifacts. For downsampling with an integer factor also see
```

"downsizing" has a specific meaning in English that does not match what we are doing here. :blush:

Reply:
One line above it also reads 'down-sizing' and two lines above, 'up-size or down-size' :grimacing: